packages/xo-web/src/common/xo/index.js

```diff
@@ -300,6 +301,12 @@ export const subscribeCurrentUser = createSubscription(() => xo.refreshUser())
 
 export const subscribeAcls = createSubscription(() => _call('acl.get'))
 
+export const subscribeHvSupportedVersions = createSubscription(
+  xoaPlans.CURRENT !== xoaPlans.SOURCES && xoaPlans.CURRENT !== xoaPlans.UNKNOWN
+    ? () => _call('xoa.getHVSupportedVersions')
+    : noop
+)
```

Comment:
Sorry, I may have changed my mind about this.

I think it would be best to avoid explicit testing in favor of handling the error.

Also, I think it does not make much sense to trigger this subscription often as the result is unlikely to change, IMHO, you should add an option to `createSubscription(cb, { polling = 5e3 } = {})`, and maybe use a long value.